proposals/NNNN-objc-implementation.md

```diff
@@ -0,0 +1,304 @@
+# Objective-C implementations in Swift
+
+* Proposal: [SE-NNNN](NNNN-objc-implementation.md)
+* Authors: [Becca Royal-Gordon](https://github.com/beccadax)
+* Review Manager: TBD
+* Status: **Awaiting implementation** (but only of resilience support)
+* Implementation: In main and release/6.0, behind the `ObjCImplementation` experimental feature flag
+* Upcoming Feature Flag: `ObjCImplementation`
```

Comment:
Remove Upcoming Feature Flag per https://forums.swift.org/t/pitch-3-objective-c-implementations-in-swift/71315/12